code/api/src/modules/userProducts/model.js

```diff
@@ -0,0 +1,26 @@
+// this tells JS or whatever that we're going to use specific variables. Google for more deets
+'use strict'
+
+module.exports = function(sequelize, DataTypes) {
+  let userProducts = sequelize.define('userProducts', {
```

Comment:
These variable names will be important to reference & confirm with the API agreement doc.